tests/Fixer/FunctionNotation/ReturnTypeDeclarationFixerTest.php

```diff
@@ -22,18 +22,47 @@
 final class ReturnTypeDeclarationFixerTest extends AbstractFixerTestCase
 {
     /**
+     * @expectedException \PhpCsFixer\ConfigurationException\InvalidFixerConfigurationException
+     * @expectedExceptionMessageRegExp #^\[return_type_declaration\] Configuration must define "space_before" being "one" or "none".$#
```

Comment:
please set expected via method instead annotation.
there are plans to drop it on phpunit 6